metis-enrichment/metis-enrichment-service/src/test/java/eu/europeana/enrichment/service/wikidata/WikidataAccessDaoTest.java

```diff
@@ -0,0 +1,79 @@
+package eu.europeana.enrichment.service.wikidata;
```

Comment:
If I manually run this test it fails..

Reply:
gdfg

Reply:
Some tests that are marked with ignore require proper configurations for the service, which are not available in the service (e.g. authorization token). We can provide the correct configurations if needed.